### PR TITLE
chore: Revamp the website content

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

### CODE REVIEW

**Comment by kushthedude** [comment]
/cc @leecalcote 
- Updated the website content.
- Announced Mentees for all programs.
- Removed the programs which are no longer useful for remaining 2020.
- Created a button for communityBridge on homepage.

**Comment by kushthedude** [comment]
Two blog posts coming right up after this!

File: _includes/footer-plain.html [hunk 1]
```diff
@@ -29,9 +29,7 @@
             <li><a class="white-text " href="/careers/internships">Internships</a>
               <ul>
               <li><a class="white-text " href="/gsoc/2020">GSoC 2020</a></li>
-              <li><a class="white-text " href="/gsod/2020">GSoD 2020</a></li>
               <li><a class="white-text " href="/communitybridge/2020">CommunityBridge 2020</a></li>
-              <li><a class="white-text " href="/mlhfellowship/2020">MLH Fellowship 2020</a></li>
```
Comment (leecalcote) [inline _includes/footer-plain.html:34]: _I think_ this is still an open possibility. I'm not sure as I think their POD program is new (?), so they might be launching more than one program (?). It'll be good to get confirmation of our rejection before removing this one.

Comment (kushthedude) [reply _includes/footer-plain.html:34]: Lets keep on hold until we hear from MLH Team.

**Comment by ashiscs** [comment]
Community Bridge icon needs to be updated !!

**Comment by kushthedude** [comment]
> Community Bridge icon needs to be updated !!

Will update once we have received an update from MLH.

**Comment by leecalcote** [comment]
Looking good!

**Pull request merged**